app/furniture/marketplace/archivable.rb

```diff
@@ -0,0 +1,30 @@
+class Marketplace
+  module Archivable
+    def self.included(model)
+      model.include(Discard::Model)
+      model.alias_method :archive, :discard
+      model.alias_method :archived?, :discarded?
+      model.alias_method :unarchived?, :kept?
+      model.alias_method :retrieve, :undiscard
```

Comment:
Should `retrieve` be `unarchive` now to be consistent?

Reply:
Probably, also if you have suggestions you can always make a PR implementing your suggestions!